app/models/manageiq/providers/network_manager.rb

```diff
@@ -54,9 +54,10 @@ class << model_name
                :class_name  => "ManageIQ::Providers::BaseManager",
                :autosave    => true
 
+    has_many :availability_zones, :primary_key => :parent_ems_id, :foreign_key => :ems_id
```

Comment:
ok, validation was not suitable because of the Nuage. So I am adding condition that it return only items with ems_id